tests/php/Authenticators/LDAPChangePasswordHandlerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\LDAP\Tests\Authenticators;
+
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\Session;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+use SIlverStripe\LDAP\Authenticators\LDAPAuthenticator;
+use SIlverStripe\LDAP\Authenticators\LDAPChangePasswordHandler;
```

Comment:
That should be use SilverStripe\LDAP\Authenticators\LDAPChangePasswordHandler; (with a lowercase i)